sdk/servicebus/service-bus/tests.yml

```diff
@@ -26,8 +27,5 @@ jobs:
         AAD_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AAD_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AAD_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        AZURE_SUBSCRIPTION_ID: $(test-subscription-id)
-        CLEAN_NAMESPACE: "true"
-        RESOURCE_GROUP: $(service-bus-test-resource-group)
-        SERVICEBUS_CONNECTION_STRING: $(service-bus-test-connection-string)
-        SERVICEBUS_CONNECTION_STRING_BROWSER: $(service-bus-test-connection-string-browser)
+        SERVICEBUS_CONNECTION_STRING: $(SERVICEBUS_CONNECTION_STRING)
+        SERVICEBUS_CONNECTION_STRING_BROWSER: $(SERVICEBUS_CONNECTION_STRING_BROWSER)
```

Comment:
A new change means we don't need to do this variable mapping anymore: https://github.com/Azure/azure-sdk-for-js/blob/master/eng/New-TestResources.ps1#L273

Reply:
@ramya0820 I understood the above comment to mean that we don't event need to have the `SERVICEBUS_CONNECTION_STRING` mapping in the tests.yml file